app/src/main/java/pl/temomuko/autostoprace/ui/staticdata/PartnersDrawables.java

```diff
@@ -11,38 +11,38 @@
  */
 public class PartnersDrawables {
 
-    private static final Integer[] DRAWABLES_ID_ARRAY = {
-            R.drawable.logo_beactive,
-            R.drawable.logo_cafe_borowka,
-            R.drawable.logo_express,
-            R.drawable.logo_grzeski,
-            R.drawable.logo_kf,
-            R.drawable.logo_kravmaga,
-            R.drawable.logo_lerni,
-            R.drawable.logo_linuxpl,
-            R.drawable.logo_mdt,
-            R.drawable.logo_partybusy,
-            R.drawable.logo_pot,
-            R.drawable.logo_profi,
-            R.drawable.logo_profi_lingua,
-            R.drawable.logo_smscenter,
-            R.drawable.logo_st,
-            R.drawable.logo_sueno,
-            R.drawable.logo_sygnet,
-            R.drawable.logo_szczypta_swiata,
-            R.drawable.logo_tarczynski,
-            R.drawable.logo_tchibo,
-            R.drawable.logo_ttwarsaw,
-            R.drawable.logo_unicar_wroclaw,
-            R.drawable.logo_unilever,
-            R.drawable.logo_zona
-    };
+    public static final List<Integer> STRATEGIC = Arrays.asList(
```

Comment:
List is mutable you are asking for troubles :D
You can use Collections.unmodifiableList() or return copy of list while making this field private